free-3.md

```diff
@@ -57,12 +90,26 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+This would probably be more of an implentation challenge. Having the graphical 
+interface be smooth and easy to use could be difficult. The language choices
+themselves should be easy since there's only a few things you can do to a boba
+drink.
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This is a fairly unambitious project in terms of the language idea itself. The 
+interface part, however, could be quite a lot of work. I don't know how well 
+Scala is suited to creating such interfaces.
+
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This could be a good project because there is a definited problem area and it 
+can't be pushed out of scope easily. However, given that the language definition
+itself won't be that complex, it may not be the best way to understand how
+to create a language.
+
```

Comment:
Absolutely a unique idea. This is a different take for sure on the common art ideas, but I wonder if this really satisfies the "language" aspect of a DSL. It seems like there would be a handful of commands that can be combined in a limited number of ways, but there wouldn't really be a way to write a fluent program in this language. It seems more like an app than a language.